src/api/AnnotationService.js

```diff
@@ -0,0 +1,150 @@
+import 'whatwg-fetch';
+import EventEmitter from 'events';
+import Annotation from '../Annotation';
+
+class AnnotationService extends EventEmitter {
```

Comment:
Should this extend from `API` as well and make use of the `makeRequest` and `axios` for consistency?

Reply:
And should it be renamed to `AnnotationAPI` like `FileVersionAPI`?

Reply:
I want to delete that file ultimately. I will split the create/edit/delete portions into one API file and the fetching of all annotations into another. That way I can stop passing the service all the way down to the threads/popover itself